pkg/mount/mountinfo_linux.go

```diff
@@ -52,8 +56,17 @@ func parseInfoFile(r io.Reader, filter FilterFunc) ([]*Info, error) {
 		p.Major, _ = strconv.Atoi(mm[0])
 		p.Minor, _ = strconv.Atoi(mm[1])
 
-		p.Root = fields[3]
-		p.Mountpoint = fields[4]
+		r, err := strconv.Unquote(`"` + fields[3] + `"`)
```

Comment:
According to kernel sources, the only characters replaced by `\nnn` are space, tab, newline, and backslash (`\`).

Initially I was opposed to using `strconv.Unquote()` but it seems if `\` is always escaped, `Unquote` can do no harm.

BTW, note that Linux file paths can indeed contain newlines, tabs, and backslashes :=0